testkit/src/lib.rs

```diff
@@ -280,7 +281,7 @@ impl TestKitBuilder {
     /// Creates the testkit.
     pub fn create(self) -> TestKit {
         if self.logger {
-            exonum::helpers::init_logger().unwrap();
+            exonum::helpers::init_logger().ok();
```

Comment:
What's the purpose of this change?

Reply:
There is a separate commit dedicated to it :slightly_smiling_face: Basically, if you try to enable the logger in an integration test (say, `counter`) via

```rust
fn init_testkit() -> (TestKit, TestKitApi) {
    let testkit = TestKitBuilder::validator()
        .with_service(CounterService)
        .with_logger()
        .create();
    // ...
}
```

then all tests except the first one will panic on this `unwrap`.